### ISSUE TITLE
Bug with cancelled/expired open orders

### ISSUE THREAD

**Issue by TechsUsInc** [bitshares/bitshares-ui#3062]
**Describe the bug**
- multiple pages of cancelled open orders in the Activity Tab
- Clicking any cancelled order tab is void of info: account, order placed, asset
- Clicking the Block number (after clicking the cancelled order tab) opens a blue "go to block" 
   field. 
- Clicking the Block number on Cryptofresh or other Explorers yields the same results.
- Multiple assets are not populating either in Portfolio balance: BEOS, Cryptobridge, 
  etc 

**To Reproduce**
1. Go to the Activity tab in the Dashboard
2. Click on a cancelled order 
3. Scroll down to your transaction if you can find it
4. Click on the Block number
5. See error of blue "go to block"
6. find the same cancelled order on a Explorer; click the block number
7. See error void of transaction information

**Expected behavior**

1. A clear and concise description of the cancelled order, transaction, and block number.
2. More information regarding cancelled orders in the Activity Tab without having to click the cancelled order tab. Clicking the cancelled order tab can still redirect to the block number, but it would be nice to simply have that information displayed, same as other transactions.  
![oo6](https://user-images.githubusercontent.com/18186497/64885414-bcbcbe00-d629-11e9-9825-60fc859e9934.JPG)

**Screenshots**
![oo3](https://user-images.githubusercontent.com/18186497/64869385-eadcd680-d606-11e9-8a7c-6c38e8b438f5.JPG)
![oo5](https://user-images.githubusercontent.com/18186497/64869391-ee705d80-d606-11e9-845a-daecee9f87a4.JPG)

**Desktop (please complete the following information):**
 - OS: Windows 8.1
 - Version bitshares 3.1.190618-rc1-rc1

** Additional information**
Pretty Print file is too large to include with this issue and will be in the following issue.

**Comment by abitmore** [comment]
Fills and cancellations are virtual operations, currently they don't show in the block explorer in bitshares-ui (see https://github.com/bitshares/bitshares-ui/issues/2719#issuecomment-513989518).

**Comment by TechsUsInc** [comment]
> Fills and cancellations are virtual operations, currently they don't show in the block explorer in bitshares-ui (see [#2719 (comment)](https://github.com/bitshares/bitshares-ui/issues/2719#issuecomment-513989518)).

Yes, I had forgotten they were virtual ops and thanks for pointing to issue #2719
